grafana_import/util.py

```diff
@@ -26,23 +28,49 @@ def load_yaml_config(config_file: str) -> ConfigType:
         raise ValueError(f"Reading configuration file failed: {ex}") from ex
 
 
-def grafana_settings(config: ConfigType, label: str) -> t.Dict[str, t.Union[str, int, bool]]:
+def grafana_settings(
+        url: t.Union[str, None],
+        config: t.Union[ConfigType, None],
+        label: t.Union[str, None]) -> SettingsType:
+    """
+    Acquire Grafana connection profile settings, and application settings.
+    """
+
+    params: SettingsType
+
+    # Grafana connectivity.
+    if url or "GRAFANA_URL" in os.environ:
+       params = {"url": url or os.environ["GRAFANA_URL"]}
+       if "GRAFANA_TOKEN" in os.environ:
+           params.update({"token": os.environ["GRAFANA_TOKEN"]})
```

Comment:
... regarding my last two comments: Better not use `from_env` for now. With the code here, we can make `--grafana_url` take precedence over `GRAFANA_URL`.